tests/kafkatest/tests/core/upgrade_test.py

```diff
@@ -112,3 +114,29 @@ def test_upgrade(self, from_kafka_version, to_message_format_version, compressio
 
         self.run_produce_consume_validate(core_test_action=lambda: self.perform_upgrade(from_kafka_version,
                                                                                         to_message_format_version))
+
+    @parametrize(from_kafka_version=str(LATEST_0_9))
+    @parametrize(from_kafka_version=str(LATEST_0_10))
+    def test_upgrade_for_cluster_id(self, from_kafka_version):
```

Comment:
Why do we need another test for this? Can we not add the asserts in the existing test?